modules/openapi-generator-gradle-plugin/README.adoc

```diff
@@ -455,3 +469,65 @@ Run with --stacktrace option to get the stack trace. Run with --info or --debug
 ----
 $ ./gradlew openApiValidate --input=/Users/jim/projects/openapi-generator/modules/openapi-generator/src/test/resources/3_0/petstore.yaml
 ----
+
+=== Generate multiple sources
+
+If you want to perform multiple generation tasks, you'd want to create a task that inherits from the `GenerateTask`.
+Examples can be found in https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator-gradle-plugin/samples/local-spec/build.gradle[samples/local-spec/build.gradle].
+
+To match the example you are using for that old swagger based plugin:
```

Comment:
Oops. I copied this from am explanation in another issue. This line needs to be removed. :x